src/components/input-field/input-field.tsx

```diff
@@ -351,6 +350,8 @@ export class InputField {
                 !!this.helperText || !!this.maxlength;
         } else {
             classList['mdc-text-field--with-leading-icon'] = !!this.leadingIcon;
+            classList['mdc-text-field--with-trailing-icon'] =
```

Comment:
Right now you’re saying that the `textarea` should never show any icons, leading or trailing. What if the consumer sends an icon to it? Have you tested if it will show up?

I think the problem was that you were getting the class for leading icon `mdc-text-field--with-leading-icon`, when the `textarea` was invalid. While you should have probably gotten the class for trailing icon instead (which opens space for the ( ! ) icon on the right side. 

I think that’s what they (MD) wants to put in there probably, which is why you were getting that class, but with a slightly wrong name (`leading` instead of `trailing`).


![image](https://user-images.githubusercontent.com/35954987/161007310-b50b3768-5dbc-4e92-8738-fc968ed7c941.png)

I don't know why the `high_importance` icon is not showing up for invalid `textarea`s right now. I'll check.

Reply:
OK, I see what happens if I remove this 👇 
<img width="449" alt="image" src="https://user-images.githubusercontent.com/35954987/161017483-68e3c2c5-86c3-40bb-a7f0-dff6ec9061c5.png">


the textarea is inside the `label` element, and fixing this requires a lot of change.
<img width="891" alt="image" src="https://user-images.githubusercontent.com/35954987/161017305-f1b64d58-ede3-4504-998b-435b4be6fd63.png">

Not worth it. So I'd say your solution is good for now

Reply:
Yeah, I don't think we've ever supported trailing icons for textareas, and I'm not even sure MDC really does… (I haven't looked it up though) 🤔